src/js/msp/MSPCodes.js

```diff
@@ -120,6 +120,12 @@ const MSPCodes = {
     MSP_TUNING_SLIDERS:             140,
     MSP_SET_TUNING_SLIDERS:         141,
 
+    MSP_CALC_PID_TUNING_SLIDERS:    142,    // calculate slider values in temp profile
+    MSP_CALC_GYRO_TUNING_SLIDERS:   143,
+    MSP_CALC_DTERM_TUNING_SLIDERS:  144,
+
+    MSP_VALIDATE_TUNING_SLIDERS:    145,    // validate slider values in temp profile
```

Comment:
Is there a reason for not using the same names as in the firmware or these?

Reply:
Fixed